src/components/learner-credit-management/data/utils.js

```diff
@@ -12,6 +13,21 @@ import {
  */
 export const transformOfferSummary = (offerSummary) => {
   if (!offerSummary) { return null; }
+  const budgetsSummary = [];
+  if (offerSummary?.budgets) {
+    const budgets = offerSummary?.budgets;
+    for (let i = 0; i < budgets.length; i++) {
+      const redeemedFunds = budgets[i].amountOfPolicySpent && parseFloat(budgets[i].amountOfPolicySpent);
+      const remainingFunds = budgets[i].remainingBalance && parseFloat(budgets[i].remainingBalance);
+      // Create an object with key-value pairs
+      const budgetEntry = {
```

Comment:
Can we use a more descriptive name here?

Reply:
> Can we use a more descriptive name here?

I have updated the name to 'updatedBudgetDetail'.